test-engineering/contract-tests/client/tests/conftest.py

```diff
@@ -6,11 +6,15 @@
 import os
 import pathlib
 from functools import lru_cache
-from typing import Dict
+from typing import Any, Dict, Set
 
 import pytest
-import requests
 import yaml
+from requests import Response as RequestsResponse
+
+import kinto
+from exceptions import MissingKintoDataFilesError
+from kinto import KintoAttachment, KintoEnvironment, KintoRecord
```

Comment:
I suggest using either the `kinto` namespace for all calls to code in that module or importing all objects we need rather than a mix of both.

Reply:
change applied: https://github.com/mozilla-services/merino/pull/409/commits/670814d39d81946a96d7d0e48d99c0e89adce4fd

I will import the objects. I'm conflicted because I like to import objects from modules, but like the clarity of the prefixes for functions.
Example:
I like `kinto.get_record(kinto_environment, record_id)` instead of `get_record(kinto_environment, record_id)`